inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -133,12 +135,22 @@ def arxiv_plot_extract(obj, eng):
             current_app.logger.exception(err)
             return
 
+        lb = LiteratureBuilder(source='arxiv', record=obj.data)
```

Comment:
@jacquerie  @david-caro What should I use for source here because the workflow object doesn't have at this point the `source` field set.